commands/docker/docker-tag.md

```diff
@@ -0,0 +1,14 @@
+### Provide new tag to docker image
+
+`docker tag {{EXISTING_IMAGE_NAME[:TAG]/ID}} {{NEW_IMAGE_NAME[:TAG]}}}`
```

Comment:
`{{EXISTING_IMAGE_NAME[:TAG]/ID}}` => `{{EXISTING_IMAGE_NAME[:TAG]}}`
Let's keep it like this and as in description below we already have mentioned either image name or id can be used

Reply:
Sure. Will update that

Reply:
Updated in the new commit